apps/backend/.env.development

```
@@ -27,7 +27,8 @@ STACK_SPOTIFY_CLIENT_SECRET=MOCK
 STACK_ALLOW_SHARED_OAUTH_ACCESS_TOKENS=true
 
 STACK_DATABASE_CONNECTION_STRING=postgres://postgres:PASSWORD-PLACEHOLDER--uqfEC1hmmv@localhost:${NEXT_PUBLIC_STACK_PORT_PREFIX:-81}28/stackframe
-STACK_DATABASE_REPLICA_CONNECTION_STRING=postgres://readonly:PASSWORD-PLACEHOLDER--readonlyuqfEC1hmmv@localhost:${NEXT_PUBLIC_STACK_PORT_PREFIX:-81}28/stackframe
+STACK_DATABASE_REPLICA_CONNECTION_STRING=postgres://postgres:PASSWORD-PLACEHOLDER--uqfEC1hmmv@localhost:${NEXT_PUBLIC_STACK_PORT_PREFIX:-81}34/stackframe
```

dotenv-linter (4.0.0)

[warning] [SubstitutionKey] The STACK_DATABASE_REPLICA_CONNECTION_STRING key is not assigned properly

(SubstitutionKey)